engine/src/main/battlecode/common/RobotType.java

```diff
@@ -200,19 +205,22 @@ public int getPassiveInfluence(int robotInfluence, int roundsAlive, int roundNum
                 return (int) Math.ceil(GameConstants.PASSIVE_INFLUENCE_RATIO_ENLIGHTENMENT_CENTER * Math.sqrt(roundNum));
             case SLANDERER:
                 if (roundsAlive <= GameConstants.EMBEZZLE_NUM_ROUNDS)
-                    return (int) (GameConstants.PASSIVE_INFLUENCE_RATIO_SLANDERER * robotInfluence);
+                    return (int) (robotInfluence *
+                            (1.0 / GameConstants.EMBEZZLE_NUM_ROUNDS +
```

Comment:
The amount a Slanderer earns back gets smaller after 100 influence; should it be changed so that it always earns back more for higher influence but the proportion it earns back slowly decreases?

https://www.desmos.com/calculator/osy49jvn1r

Reply:
You've made a mistake in your formula; there's a 0.01 which should be 0.001.
With the current value, the diminishing returns start at 949